packages/amplify-e2e-tests/src/__tests__/auth_2.test.ts

```diff
@@ -97,7 +99,7 @@ describe('amplify add auth...', () => {
     expect(lambdaFunction.Configuration.Environment.Variables.GROUP).toEqual('mygroup');
   });
 
-  it('...should allow the user to add auth via API category, with a trigger and function dependsOn API', async () => {
+  it.only('...should allow the user to add auth via API category, with a trigger and function dependsOn API', async () => {
```

Comment:
Did you mean to leave this? FYI, you can use Jest's `-t` flag to only run specific tests.